src/Analysis/Codelyzer.Analysis.Model/MethodBlock.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace Codelyzer.Analysis.Model
+{
+    public class MethodBlock : BaseMethodDeclaration
```

Comment:
Just making sure this is meant to be BaseMethodDeclaraction instead of the MethodBlockBase that was added below